README.md

```diff
@@ -46,6 +49,9 @@ Total: 10, Dialing: 1, Established: 9, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
 ```
+Unsuccessful execution:
+```bash
```

Comment:
??

Reply:
There was info in the latest commit... in addition, i elaborated a little bit more the line introducing it